internal/commands/ip_access/create.go

```diff
@@ -0,0 +1,52 @@
+package ipaccess
+
+import (
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cli/user"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/spf13/pflag"
+)
+
+// CommandMetaCreate is the command meta for the `ip-access create` command
+var CommandMetaCreate = cli.CommandMeta{
+	Use:     "create",
+	Aliases: []string{"add"},
```

Comment:
no other "create" command yet uses `"add"` as an alias, is this coming from product or somewhere?  if so, we should standardize that across all the "create" commands, not just here

Reply:
This is what was decided on the spec (for now). @christineschen tagging you for visibility